test/test_circular_buffer.jl

```diff
@@ -5,7 +5,8 @@ using Base.Test
 cb = CircularBuffer{Int}(5)
 @test length(cb) == 0
 @test capacity(cb) == 5
-@test_throws ArgumentError first(cb)
+# throws ArgumentError on v0.4 and BoundsError on v0.5
+@test_throws Exception first(cb)
```

Comment:
do you know what the version cutoff for this change was? might be better to be more specific